### ISSUE TITLE
Encountering a `Schema is missing its top node type" ('doc')` error from example in README

### ISSUE THREAD

**Issue by dslatkin** [marekdedic/prosemirror-unified#160]
Hi, thanks for making this library!

I'm attempting to try it out with the example from the README, but I'm getting the following error:

```
index.js:2246 Uncaught RangeError: Schema is missing its top node type ('doc')
    at _NodeType.compile (index.js:2246:19)
    at new Schema (index.js:2372:35)
    at SchemaBuilder.build (SchemaBuilder.ts:27:12)
    at new ProseMirrorUnified (ProseMirrorUnified.ts:28:60)
    at main.ts:9:13
```

I'm using Vite and ProseMirror seems to work fine when I use it without `prosemirror-unified`. I've tried to copy the example as closely as possible but can't figure out why this is happening. You should be able to recreate the issue using this repo:

https://github.com/dslatkin/prosemirror-unified-issue

There's a dev container definition in it too if you have Docker and the VS Code extension. If you run `npm run dev` and visit the URL you should see the console error. [This commit](https://github.com/dslatkin/prosemirror-unified-issue/tree/5de0b18dd6e9135bec2989c733e580d99d607770) has the working basic ProseMirror example.

Thank you for any help!

**Comment by marekdedic** [comment]
Hi,
hmm, that's strange, will take a look.

Thanks for the report!

**Comment by marekdedic** [comment]
Fixed by marekdedic/prosemirror-remark#154

**Comment by dslatkin** [comment]
Hi @marekdedic, I just tested. It looks like pinning my `prosemirror-unified` to `0.5.0` fixed it temporarily. But after marekdedic/prosemirror-remark#154, if I run `npm update` in the example repo [dslatkin/prosemirror-unified-issue](https://github.com/dslatkin/prosemirror-unified-issue), I still have the same error. I also tried `rm -fr node_modules && npm install` just to make sure there wasn't some weird dependency thing going on and that didn't work, either. 😕
